packages/ringcentral-widgets/components/ActiveCallPanel/ConferenceInfo.js

```diff
@@ -1,14 +1,110 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import debounce from 'ringcentral-integration/lib/debounce';
 
 import CallAvatar from '../CallAvatar';
 import dynamicsFont from '../../assets/DynamicsFont/DynamicsFont.scss';
 import styles from './styles.scss';
 import i18n from './i18n';
 
+const MAXIMUM_AVATARS = 4;
+
+const WIDTH_PER_AVATAR = parseInt(styles.conferenceAvatarSize, 0); // 51
+
+const AVATAR_MERGIN_LEFT = parseInt(styles.avatarMerginLeftSize, 0); // -20
+const PEDDING_WIDTH = parseInt(styles.avatarPaddingSize, 0); // 15
+
+const minWidthCalculator = count =>
+  (WIDTH_PER_AVATAR * count) +
+  AVATAR_MERGIN_LEFT * (count - 1) + (PEDDING_WIDTH * 2) + 1 + 2;
+
+// when the container width reachs below item of width, display the avatar amount of count.
+const KINDS_OF_WIDTH_THAT_NEED_ADAPATER = [
+  { avartarCount: 0, width: minWidthCalculator(2), },
```

Comment:
we show at least 1 avatar

Reply:
Confirmed with Albert, we could show it like that when the window is at <100 widths.